setup.py

```diff
@@ -57,6 +57,10 @@ def run(self):
                 cwd=cwd,
             )
 
+        # Generate bash autocompletion scripts
+        check_call('register-python-argcomplete virtme-ng > virtme-ng-prompt', shell=True)
+        check_call('register-python-argcomplete vng > vng-prompt', shell=True)
```

Comment:
on [Debian side](https://salsa.debian.org/debian/virtme-ng/-/blob/debian/debian/rules?ref_type=heads), they also try to use `register-python-argcomplete3`. Maybe safer to try as well if the previous one is not available?

Reply:
We know for sure it's available because we have a dependency on argcomplete.